server/src/apub/mod.rs

```diff
@@ -138,37 +149,85 @@ fn create_tombstone(
       tombstone.set_deleted(convert_datetime(updated).into());
       Ok(tombstone)
     } else {
-      Err(format_err!(
-        "Cant convert to tombstone because updated time was None."
-      ))
+      Err(format_err!("Cant convert to tombstone because updated time was None.").into())
     }
   } else {
-    Err(format_err!(
-      "Cant convert object to tombstone if it wasnt deleted"
-    ))
+    Err(format_err!("Cant convert object to tombstone if it wasnt deleted").into())
   }
 }
 
+#[async_trait::async_trait(?Send)]
```

Comment:
the ?Send bound is to prevent generation of traits that with `fn ... -> Pin<Box<dyn Future<..> + Send>>` signatures, which don't play nice with Actix Web, where a lot of things are !Send (like http clients)